packages/vscode-extension/e2e_tests/tests/syntaxValidation.spec.ts

```diff
@@ -114,7 +114,7 @@ suite('Syntax validation spec', () => {
     });
   });
 
-  test.only('Correctly validates a non cypher file when selecting cypher language mode', async () => {
```

Comment:
This was an oversight I got merged by mistake 🙈

Reply:
it happens 😅